torchaudio/compliance/kaldi.py

```diff
@@ -512,3 +514,172 @@ def fbank(
         mel_energies = mel_energies - col_means
 
     return mel_energies
+
+
+def get_LR_indices_and_weights(orig_freq, new_freq, output_samples_in_unit, window_width,
```

Comment:
Is this exposed to the user? This function name seems fine to me, but I just want to make sure we consciously agree this is a consistent capitalization/convention if so.

Reply:
I don't think this needs to be exposed for now. We should prefix it with an underscore to indicate that it's internal.